tests/test_wps_split_merged_climos.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+
+from pywps import Service
+from pywps.tests import assert_response_success
+
+from .common import client_for, TESTDATA
+from thunderbird.processes.wps_split_merged_climos import SplitMergedClimos
+
+gcm_360_climos_local = TESTDATA["test_local_gcm_360_climos_nc"]
+gcm_climos_local = TESTDATA["test_local_gcm_climos_nc"]
+tasmax_climos_local = TESTDATA["test_local_tasmax_climos_nc"]
+gcm_360_climos_opendap = TESTDATA["test_opendap_gcm_360_climos_nc"]
+gcm_climos_opendap = TESTDATA["test_opendap_gcm_climos_nc"]
+tasmax_climos_opendap = TESTDATA["test_opendap_tasmax_climos_nc"]
+
+client = client_for(Service(processes=[SplitMergedClimos()]))
+
+
+def check_success(datainputs):
+    resp = client.get(
+        service="wps",
+        request="Execute",
+        version="1.0.0",
+        identifier="split_merged_climos",
+        datainputs=datainputs,
+    )
+    assert_response_success(resp)
+
+
+def run_single_file(netcdf):
+    datainputs = ("netcdf=@xlink:href={};").format(netcdf)
+    check_success(datainputs)
+
+
+def run_multiple_files(netcdf):
+    datainputs = ""
+    for i in range(len(netcdf)):
+        datainputs += "netcdf=@xlink:href={};".format(netcdf[i])
+    check_success(datainputs)
```

Comment:
While this works, it may be made more clear by changing some of the names around.  Instead of using the index why not just loop through each `netcdf`?

```python3
def run_multiple_files(netcdfs):
    datainputs = ""
    for netcdf in netcdfs:
        datainputs += f"netcdf=@xlink:href={netcdf};"
    check_success(datainputs)